pkg/user/tx_client_test.go

```diff
@@ -335,9 +337,8 @@ func TestEvictions(t *testing.T) {
 		t.Skip("skipping evictions test in short mode")
 	}
 
-	ttlNumBlocks := int64(1)
-	blocksize := int64(1048576) // 1 MiB
-	_, txClient, ctx := setupTxClient(t, ttlNumBlocks, blocksize)
+	blocksize := int64(1024 * 1024 * 2) // 2 MiB
+	_, txClient, ctx := setupTxClient(t, 1, 64, blocksize)
```

Comment:
see #6091 for more details

using a single MB block limit but a larger square limit results in a chain halt. 

The previous test never hit this due to the mempool not working and only getting a single tx at a time. TxClient will try keep retrying over and over until its gets included.

@ninabarbakadze is the txclient supposed to loop continuously? I forget the answer here as iirc you mentioned it shouldn't be doing that. Is this only for evictions?